ros/angel_system_nodes/setup.py

```diff
@@ -33,6 +33,7 @@
                 'annotation_event_monitor = angel_system_nodes.annotation_event_monitor:main',
                 'object_detector_with_descriptors = angel_system_nodes.object_detector_with_descriptors:main',
                 'uho_activity_detector = angel_system_nodes.uho_activity_detector:main',
+                'hmm = angel_system_nodes.hmm:main',
```

Comment:
This name is confusing and lacks context. I would more recommend that we rename the existing task monitor to something like `task_monitor_v1` and this `task_monitor_v2` as they fulfill the same role (to my understanding). Of course, reflect that name change to the V1 node in the existing config files to maintain relevance.

It would be good to start a .rst file under `docs/task_monitor.rst` that lists the relevant nodes and a short description about what the difference is (e.g. v1 is the first iteration that combined activity classifications and configuration to very simply make task step determinations, and v2 uses a hidden markov model to deduce step transitions). This initial doc doesn't need to be more detailed than that, we just need to begin tracing history before we lose track of it.